diagnostic.md

```diff
@@ -5,27 +5,27 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    A profile on a company website. A person's personal info, image, summary, and the like could all be components. This is because they would be goot to have as templates that will not be linked to directly, but rather as a part of a view. This allows all profiles to use the same component templates.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember generate component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    a component is generated with a template.hbs and component.js file. I beleive test files are also added and modified.
```

Comment:
Yep! This question also asks about the responsibilities of those files. 

Template files (`template.hbs`) are responsible for containing markup
(defining structure), binding data, and binding and dispatching actions.

Components objects (`component.js`) are responsible for defining actions,
declaring HTML attributes, and other presentation or behavior.

In general, templates are responsible for markup, while components are
responsible for presentation and behavior.